plugin.audio.podcatcher/simplexbmc.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+#-------------LicenseHeader--------------
+# plugin.video.podcatcher- A plugin to organise Podcasts
+# Copyright (C) 2010  Raptor 2101 [raptor2101@gmx.de]
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import xbmc, xbmcgui, xbmcplugin,xbmcaddon, sys, re, time
+__plugin__ = "PodCatcher"
+
+regex_decimal = re.compile("\\d+");
+regex_duration = re.compile("(\\d+:){0,2}\\d+");
+settings = xbmcaddon.Addon(id='plugin.audio.podcatcher')
+translation = settings.getLocalizedString
+
+class SimpleXbmcGui(object):
+  def __init__(self,path):
+    self.path = path;
+
+  @staticmethod
+  def log(msg):
+    if not isinstance(msg,str) and not isinstance(msg,unicode):
+      xbmc.log("[%s]: %s" % (__plugin__, type(msg)))
+    else:
+      xbmc.log("[%s]: %s" % (__plugin__, msg.encode('utf8')))
+
+  def buildMediaItem(self, element, markUnread):
+    if(element.subTitle == ""):
+      title = "%s (%s)"%(element.title,element.duration);
+    else:
+      title = "%s - %s (%s)"%(element.title,element.subTitle,element.duration);
+
+    if(type(element).__name__ == 'FeedItem'):
+      title = "[%s] %s"%(time.strftime("%d.%m",element.date),title);
+
+    if(markUnread and not element.readed):
+      title = "(*) %s"%(title);
+    else:
+      title = "%s"%(title);
+    self.log(element.picture)
+    if(element.picture is not ""):
+      liz=xbmcgui.ListItem(title, iconImage="DefaultFolder.png", thumbnailImage=element.picture)
```

Comment:
Please remove iconImage and thumbnailImage kwargs and use liz.setArt({'icon':..., 'thumb':...})